README.adoc

```diff
@@ -42,37 +45,55 @@ and configure a _Run/Debug configuration_ like this:
 * Working directory: `$MODULE_DIR$` (must resolve to optaplanner-examples directory)
 * Use classpath of module: `optaplanner-examples`
 
-=== Starter issues
+### Starter issues
 
 If you're just starting out with OptaPlanner and want to contribute,
 take a look at our https://issues.redhat.com/issues/?jql=project%20%3D%20PLANNER%20AND%20status%20in%20(Open%2C%20Reopened)%20AND%20labels%20%3D%20starter%20ORDER%20BY%20priority%20DESC[starter issues].
 They're specifically chosen to be easier for first time contributors.
 
-== Developing Drools, OptaPlanner and jBPM
+## Contributing to OptaPlanner
+
+This is an open source project, and you are more than welcomed to contribute!:heart:
+
+### Getting into
 
-*If you want to build or contribute to a kiegroup project, https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/main/README.md[read this document].*
+Working on a contribution or want to start an opinionated discussion? +
+Join our https://groups.google.com/g/optaplanner-dev[discussion] or mail directly through optaplanner-dev@googlegroups.com
 
-*It will save you and us a lot of time by setting up your development environment correctly.*
-It solves all known pitfalls that can disrupt your development.
-It also describes all guidelines, tips and tricks.
-If you want your pull requests (or patches) to be merged into main, please respect those guidelines.
+Want to submit an issue, then check out our bord on https://issues.redhat.com/projects/PLANNER/issues[Jira]
 
-=== Code style
+### Time to make a change?
+Every change must go through *Pull Request*. Please follow the <<Contributing>> so the feature/bugfix and it's tests could be reviewed. OptaPlanner has CLI that run checks against your branch after PR is suggested. So don't worry if something gets wrong or breaks other parts of functionality our friendly PR maintainers will help you with that.
 
-OptaPlanner has adopted the https://github.com/quarkusio/quarkus[Quarkus] code style, enforces it, and automatically formats code during the build.
-To setup your IDE, please see the
-<<build/optaplanner-ide-config/ide-configuration.adoc#, IDE Setup Instructions>>.
+### Contributing
```

Comment:
Code style of adoc files: Use = instead of # for titles
Asciidoctor supports # too, for MarkDown compatibility, but we always use =

```suggestion
=== Contributing
```

Reply:
FIxed